src/components/banner/styles/Banner.scss

```diff
@@ -0,0 +1,92 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+.banner {
+    background: linear-gradient(45deg, $mirage 0%, $science-blue 100%);
+    box-shadow: $banner-box-shadow;
+    padding: $banner-padding;
+    height: $banner-height;
+    box-sizing: border-box;
+    position: relative;
+
+    button {
```

Comment:
dont use the element directly, add a class on it

Reply:
I don't have access to it, it's the one sent with the content. It is scoped to the banner context.

Reply:
If its sent in the content, I say let the content style itself. Nothing more frustrating then syles overriding each other.

Reply:
You could front this style by putting it in a class and front it as static variable of the Banner component, then the user of the component could choose wether or not to use the class in its buttons

Reply:
You're right. At first it was because I found an old mockup with special buttons, but since they're "normal" now anyway, better not style them. I will remove.